src/cli/repl.rs

```diff
@@ -563,14 +597,61 @@ impl Repl<F> {
             }
             "verify" => {
                 let first = self.peek1(cmd, args)?;
-                match self.store.fetch_string(&first) {
-                    None => bail!(
-                        "Proof ID {} not parsed as a string",
-                        first.fmt_to_string(&self.store)
-                    ),
-                    Some(proof_id) => {
-                        LurkProof::verify_proof(&proof_id)?;
+                let proof_id = self.get_string(&first)?;
+                LurkProof::verify_proof(&proof_id)?;
+            }
+            "defpackage" => {
+                // TODO: handle args
+                let (name, _args) = self.store.car_cdr(args)?;
+                let name = match name.tag {
+                    ExprTag::Str => self.state.borrow_mut().intern(self.get_string(&name)?),
+                    ExprTag::Sym => self.get_symbol(&name)?.into(),
+                    _ => bail!("Package name must be a string or a symbol"),
+                };
+                println!("{}", self.state.borrow().fmt_to_string(&name));
+                let package = Package::new(name);
+                self.state.borrow_mut().add_package(package);
+            }
```

Comment:
It goes without saying that `defpackage` will need to be made more powerful. It might be worth creating at least the stub for this future work by defining it as a real function that can be called here.

Reply:
I've opened an issue for it